build.rs

```diff
@@ -20,8 +20,27 @@ fn main() {
     env::var("PROFILE").unwrap()
   };
 
+  // Equivalent to target arch != host arch
+  let is_different_target_arch =
```

Comment:
Would be useful to list some examples of what CARGO_CFG_TARGET_ARCH and HOST looks like.